packages/framework/src/Resources/config/services_test.yml

```diff
@@ -234,9 +212,14 @@ services:
         arguments:
             $indexPrefix: 'test-%env(ELASTIC_SEARCH_INDEX_PREFIX)%'
 
+    Elasticsearch\Client:
```

Comment:
Why are these changes here? Maybe I'm just surprised because the change is in the commit 86abff5780

Reply:
my fault during fixes for the tests i will remove it